### PR TITLE
prep-css-flexbox - Bryan Brown

### PR DESCRIPTION
**Pull request by bbrown949**
Screenshot 2023-01-13 at 7 04 19 PM
Screenshot 2023-01-13 at 7 03 55 PM
Screenshot 2023-01-13 at 7 03 47 PM
Screenshot 2023-01-13 at 7 03 39 PM

### CODE REVIEW

File: prep-css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck{display:flex; min-height: 100vh; align-items:center; justify-content:center;}
```
Comment (Zhiwutian) [inline prep-css-flexbox/align-items.html:9]: Hey Bryan, I would recommend formatting you CSS like this
```css
    #card-deck {
      min-height: 100vh;
      display: flex;
      align-items: center;
      justify-content: center;
    }
```
as it makes it much easier to read for other developers and allows for lots of properties without having to scroll to the right in the code.

good work on the functionality!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Hey Bryan, functionality is solid, just make sure to format your css as noted.

File: prep-css-flexbox/align-items.html (hunk 1 above)
Comment (bbrown949) [reply prep-css-flexbox/align-items.html:9]: copy that thanks

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**